html/elements/input/week.json

```diff
@@ -0,0 +1,57 @@
+{
+  "html": {
+    "elements": {
+      "input": {
+        "input-week": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/html/Element/input/week",
+            "support": {
+              "chrome": {
+                "version_added": "20"
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": "12"
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": false,
+                "notes": "This feature is not implemented yet. See <a href='https://bugzil.la/888320'>bug 888320</a> and <a href='https://wiki.mozilla.org/TPE_DOM/Date_time_input_types'>TPE DOM/Date time input types</a>."
```

Comment:
This is `false` for both desktop and Android Firefoxes and @wbamberg's [comment]([https://github.com/mdn/browser-compat-data/pull/2200#discussion_r197279506]) on #2200 applies to both as well